conda/pytorch-nightly/meta.yaml

```diff
@@ -9,6 +9,8 @@ requirements:
   build:
     - cmake
     - {{ compiler('c') }} # [win]
+
+  host:
```

Comment:
Is this change necessary for the Windows pipeline? Why's it needed?

Reply:
According to the conda-build [docs](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host), once the ` {{ compiler() }}` jinja2 functions are used, there should be a host section. The reason that we use the aforementioned jinja2 function is that we have to override the default compiler. I just wonder why it doesn't work on Linux, so I'll try to add a Linux job first.

Reply:
@pjh5 From the logs [here](https://peter-jc.visualstudio.com/pytorch_builder/_build/results?buildId=99&view=logs), it seems that the script works on Linux. Is there anything missing?

Reply:
@pjh5 Plus: The config for the Linux job: https://github.com/pytorch/builder/pull/191/files#diff-f6392e6105a578ca4622abcfc34d2834R160.

Reply:
I just remember that this change broke the nightlies last time it was introduced. I'll run a job with these changes to make sure they still work

Reply:
Okay it seems that this works okay